### PR TITLE
added the bulk generation of promotion keys

### PR DESCRIPTION
**Pull request by peymanmortazavi**

### CODE REVIEW

File: app/promotion_keys.py [hunk 1]
```diff
@@ -25,10 +25,11 @@ def post(self):
         if showtime is None:
             return self.error(404, "Show time not found.")
 
-        promotion_key = yield create_promotion_key(showtime_id)
-        self.set_header("Location", "{0}/api/promotion_keys/{1}".
-                        format(CONFIG.get("base_url"), promotion_key))
-        self.api_response({'promotion_key': promotion_key}, 201)
+        promotion_keys = []
+        for i in range(0, count):
+            promotion_keys.append((yield create_promotion_key(showtime_id)))
```
Comment (mynameisfiber) [inline app/promotion_keys.py]: Not something you need to fix at all, but just so you know, you can do this:

```
promotion_keys = yield [create_promotion_key(showtime_id) for _ in range(count)]
```

You also don't need to provide the lower bound on a `range` unless it's _not_ 0.

**Comment by peymanmortazavi** [comment]
Ah sweet. I'm starting to like Python :) thanks for the info.

File: app/promotion_keys.py [hunk 2]
```diff
@@ -16,6 +15,7 @@ class PromotionKeysHandler(BaseHandler):
     @gen.coroutine
     def post(self):
         showtime_id = self.get_argument('showtime_id', None)
```
Comment (mynameisfiber) [inline app/promotion_keys.py:17]: If we are going to have `showtime_id` be the convention, could you change [app/showtimes.py:26](https://github.com/QuantifiedSelfless/quantifiedSelf/blob/improvement/bulk-promo/app/showtimes.py#L26) to also use this?

**Comment by mynameisfiber** [comment]
@peymanmortazavi One note about api argument standardization... other than that LGTM :+1: :hammer: :hamburger: :helicopter:

**Pull request merged**